tests/unit_tests/sensitivities/dRdX_fd_vs_ad.cpp

```diff
@@ -251,7 +258,11 @@ int main (int argc, char * argv[])
                     }
                 }
 
-                if ((*pde==PDEType::euler) || (*pde==PDEType::navier_stokes)) {
+                if ((*pde==PDEType::euler) || (*pde==PDEType::navier_stokes)
+#if PHILIP_DIM==3
+         || ((*pde==PDEType::physics_model) && (all_parameters.model_type==ModelType::large_eddy_simulation))
```

Comment:
I'm going to guess that `all_parameters.model_type=ModelType::large_eddy_simulation` by default through declare_parameters/parse_parameters. I would still set it manually somewhere up there, just in case we end up adding more model types.

Reply:
fixed